lib/commands.py

```diff
@@ -110,9 +110,8 @@ def _ssh(hostname_or_ip, cmd, check=True, simple_output=True, suppress_fingerpri
     stdout = []
     for line in iter(process.stdout.readline, b''):
         readable_line = line.decode(errors='replace').strip()
-        if readable_line != '':
-            stdout.append(line)
-            OUPUT_LOGGER.debug(readable_line)
+        stdout.append(line)
+        OUPUT_LOGGER.debug(readable_line)
```

Comment:
Maybe we'll want to skip the empty line for the logger, while appending it to stdout.

If we started skipping empty line, this is probably because it made the logs less readable in some cases. Or maybe there was a more important reason? Should we go back on this decision? We'll need to run various tests with debug output enabled to determine this.

Reply:
I think the `OUTPUT_LOGGER` line should be put back in the if as indeed there's no point in logging empty lines.
As for the stdout, I guess we need to run various tests indeed to be sure we didn't break other tests not expecting empty lines in the result.

Reply:
Except to log the "reality". For example in the case of the key from varstored you may ask "ok there is something missing". I mean if it doesn't pollute logs too much I prefer log it but I'm ok to put it back in the IF since your are 2 against 1 in favor to not log it :)

Reply:
We need to see what the logs looks like as you propose, if its too much spacing or not.

Reply:
I ran many tests, and haven't seen adverse effects.